pkg/common/unittestcommon/utils.go

```diff
@@ -212,6 +212,16 @@ func (f *fakeVolumeOperationRequestInterface) StoreRequestDetails(
 	return nil
 }
 
+// DeleteRequestDetails deletes the VolumeOperationRequestDetails for the given
+// name, if any, stored by the fake VolumeOperationRequest interface.
+func (f *fakeVolumeOperationRequestInterface) DeleteRequestDetails(
+	ctx context.Context,
+	name string,
+) error {
+	delete(f.volumeOperationRequestMap, name)
```

Comment:
is object locking handled internally?

Reply:
Not necessary. These are just fake implementations in unittest code.